charts/iam-manager/templates/namespace.yaml

```diff
@@ -3,4 +3,6 @@ kind: Namespace
 metadata:
   labels:
     {{- include "iam-manager.labels" . | nindent 4 }}
+    pod-security.kubernetes.io/enforced: baseline
+    pod-security.kubernetes.io/warn: baseline
```

Comment:
@diranged It looks like this chart might have been created during our k8s infancy. I am happy to start a micro design meeting to re-up this chart to the latest standards if you want instead of hot patching like this.

Reply:
this is fine for now...